aws-sso-permissionset/aws-sso-permissionset.json

```diff
@@ -85,7 +85,10 @@
     },
     "InlinePolicy": {
       "description": "The inline policy to put in permission set.",
-      "type": "string",
+      "type": [
+        "object",
+        "string"
+      ],
       "pattern": "[\\w+=,.@-]+",
```

Comment:
qq. Does pattern is still applicable property. if type can be either ?

Reply:
Good catch, pattern may not be applied to the combined type. Will remove it and let runtime validation catch it.